packages/migrations/src/migration.ts

```diff
@@ -184,14 +195,16 @@ export async function runMigrationsAsync(
         zrxAssetData,
     );
 
-    // DutchAuction
-    const dutchAuction = await wrappers.DutchAuctionContract.deployFrom0xArtifactAsync(
-        artifacts.DutchAuction,
-        provider,
-        txDefaults,
-        artifacts,
-        exchange.address,
-    );
+    // TODO(fabio): Uncomment dutchAuction once the @0x/contracts-extensions is refactored
```

Comment:
bump `dummyTransactionCount` to `6` to avoid changing all contract-addresses that follow. Once the dust has settled we'll remove this hack and ensure all of our documentation and examples reference the correct contracts for the Ganache migration.